textattack/shared/utils/tensor.py

```diff
@@ -11,7 +11,7 @@ def batch_model_predict(model_predict, inputs, batch_size=32):
     i = 0
     while i < len(inputs):
         batch = inputs[i : i + batch_size]
-        batch_preds = model_predict(batch)
+        batch_preds = model_predict(batch)[0]
```

Comment:
Please revert this as well. This tuple unwrapping happens in `HuggingFaceModelWrapper`: https://github.com/QData/TextAttack/blob/master/textattack/models/wrappers/huggingface_model_wrapper.py#L48

Reply:
Reverted